### DISCUSSION TITLE
List resource from table builder when entry has on "active" field on false

### DISCUSSION THREAD

**Discussion by jechazelle** [filamentphp/filament#14208] [Help]
### Package

Table builder

### Package Version

v3.2.111

### How can we help you?

Hi,

I would like to know how can I list the resource with "active" field set on **true** AND **false**.

Currently, I have a table with "active" field. When a switch the entry with "active" field on false, I don't see it on listing resource:

<img width="1144" alt="Capture d’écran 2024-09-11 à 17 02 55" src="https://github.com/user-attachments/assets/ce09daf2-31a6-4744-96d6-24f033942176">

I tried with "is_active" field, and I always see the resource.

Is Filament configured to not display entries with an "active" field set to false?

If so, how can I still display entries with a false status on "active"

**Comment by leandrocfe** [comment]
_Is Filament configured to not display entries with an "active" field set to false_

No. Probably something in your code that applies this.
